### ISSUE TITLE
Creative failures and player actions

### ISSUE THREAD

**Issue by andmig** [InteractiveAdvertisingBureau/SIMID#312]
Current SIMID spec recommends that the player abandons the ad altogether if the creative fails to meet certain logic/requirements. Exceptions are made for SSAI and live streams when the player cannot drop the media playback. 

Specifically, such recommendations are made for:

1. Creative failure to respond to the `Player;init` within the allotted timeout.
1. Creative failure to respond to the `Player:startCreative` before a timeout expires. 
1. Creative posting `Creative:fatalError` before the media starts.
1. Proposed: creative failure to initialize a session - post `createSession` message.

The spec may benefit from the simplification of creative failure handling. SIMID may recommend timeout but never that the player abandons the ad. Once timeouts lapse - the player proceeds to the next step(s).

In other words, once the player loads the creative it is free to utilize the impression no matter the creative state.

For example:
1.  If the creative fails to initialize a session - the player proceeds with the media playback while still waiting for the creative to send the createSession message. 

1. In case of the creative delay/failure to respond to the `Player:init`, the player proceeds to the media playback. Should creative post a response to the `Player:init` at a later point - the player will pick up communication from where it was left.

**Comment by ryanthompson591** [comment]
Discussed with Andre.
1. We think the amount of time the player should wait for the creative to respond should be buffering time of a video. This means the player can start buffering the video right away and there's no delays.
2. If the creative doesn't initialize the session, respond to start or respond to init--- The player "may" play the video.  This was considered to be an edge case and shouldn't happen often.  Likely all players will play the video in this case, but there may be players that want to abandon the ad.  Since this is such an edge case, its ok to allow the player to decide.
